tests/transformer/test_assign.py

```diff
@@ -0,0 +1,59 @@
+from RestrictedPython._compat import IS_PY2
+from RestrictedPython.Guards import guarded_unpack_sequence
+from tests import e_exec
+
+import pytest
+
+
+@pytest.mark.parametrize(*e_exec)
+def test_RestrictingNodeTransformer__visit_Assign(e_exec, mocker):
```

Comment:
As there is another test function in this module for `visit_Assign` its name should end in `__1`.